src/xrCore/LocatorAPI.cpp

```diff
@@ -612,7 +612,7 @@ void CLocatorAPI::ProcessOne(pcstr path, const _finddata_t& entry)
             return;
         if (0 == xr_strcmp(entry.name, ".."))
             return;
-        if(path[xr_strlen(path) - 1] != _DELIMITER || path[xr_strlen(path) - 1] != '/')
+        if(path[xr_strlen(path) - 1] != _DELIMITER && path[xr_strlen(path) - 1] != '/')
```

Comment:
Let's fix the formatting just by the way...
```suggestion
        if (path[xr_strlen(path) - 1] != _DELIMITER && path[xr_strlen(path) - 1] != '/')
```